src/PrestaShopBundle/Resources/config/services/core/grid/grid_definition_factory.yml

```diff
@@ -91,16 +91,20 @@ services:
     prestashop.core.grid.definition.factory.language:
         class: 'PrestaShop\PrestaShop\Core\Grid\Definition\Factory\LanguageGridDefinitionFactory'
         parent: 'prestashop.core.grid.definition.factory.abstract_grid_definition'
-        arguments:
-            - "@=service('router').generate('admin_common_reset_search', {'controller': 'language', 'action': 'index'})"
-            - "@=service('router').generate('admin_languages_index')"
         public: true
 
     prestashop.core.grid.definition.factory.currency:
         class: 'PrestaShop\PrestaShop\Core\Grid\Definition\Factory\CurrencyGridDefinitionFactory'
         parent: 'prestashop.core.grid.definition.factory.abstract_grid_definition'
         public: true
 
+    prestashop.core.grid.definition.factory.currency_for_language:
```

Comment:
We dont need this anymore 😉(code dedicated to test the behavior)

Reply:
removed